### DISCUSSION TITLE
Bulk import model instances

### DISCUSSION THREAD

**Discussion by desmondhoward** [wpengine/atlas-content-modeler#450] [Q&A / Help / Support] [answered]
I understand that you can do an import of the model/field definitions. 
Is there a way to bulk import instances of a model instead of adding them one by one so that they appear in the table?
<img width="539" alt="Screen Shot 2022-02-26 at 4 50 25 PM" src="https://user-images.githubusercontent.com/96273496/155860229-7350815d-f209-449b-ba4a-ee09c1253d4c.png">

**Comment by nickcernis** [comment, marked as answer]
Hi, @desmondhoward!

If you're comfortable using WP-CLI, we have a feature called “Blueprints” that lets you export and then import model entries. It's documented here:

https://github.com/wpengine/atlas-content-modeler/blob/main/docs/blueprints/index.md
https://github.com/wpengine/atlas-content-modeler/blob/main/docs/wp-cli/index.md

We hope to add the ability to do this via a web UI on the Import/Export Tools admin page, but I don't have a timeline for that work just yet.
